pages/thc/[processRef]/loading.tsx

```diff
@@ -391,14 +391,48 @@ const useFetchTenancyData = (): UseApiWithStorageReturn<
   });
 };
 
+const useOfficerData = (): UseApiWithStorageReturn<
+  ExternalDatabaseSchema,
+  "officer"
+> => {
+  const router = useRouter();
+
+  const processRef = getProcessRef(router);
+  const data = getMatApiData(processRef);
+
+  return useApiWithStorage({
+    endpoint: "/v1/officer",
+    query: { data },
+    jwt: {
+      sessionStorageKey:
+        isClient && processRef ? `${processRef}:matApiJwt` : undefined,
```

Comment:
Ideally, let's replace this with the helper `getMatApiJwt`.

Reply:
`getMatApiJwt` returns the JWT itself, whereas this is setting the key. Unless I'm misunderstanding, and you meant I should create another helper to do this?

Reply:
No wait. You're right. Ignore me.